### ISSUE TITLE
Documentation about training NLP models from scratch

### ISSUE THREAD

**Issue by li-wei85** [tensorflow/models#10074]
The [train.md](https://github.com/tensorflow/models/blob/master/official/nlp/docs/train.md) file states that Documentation about PreTraining Language model from scratch is coming "soon". 

As I find it very hard to launch a pre-training task I wanted to know if there was any examples on how to do it for a pre-training of electra or bert for example ?

I saw an Electra Pretraining task was written lately. I wanted to know if I could reproduce the original Electra results with that one.

Sincerely

Li

**Comment by saikumarchalla** [comment]
@li-wei85  Regrading the electra,  Could you please post your query in [google-research](https://github.com/google-research/electra) repo. Also, Please check this [article](https://towardsdatascience.com/understanding-electra-and-training-an-electra-language-model-3d33e3a9660d) for more information about the model.Thanks

**Comment by saberkun** [comment]
Thanks for reporting the concerns. Yes, it is our job to add the doc. Will do.

For Electra, yes, it can reproduce the pretraining and on finetuning metrics, we can make the paper results for small and base. The original electra finetuning added per-layer learning rate adjustment and we feel it may help the larger and deeper model quality.

Overall, we have a paper published based on the Electra baseline here. We will update the information. 

Sorry for the delay.

**Comment by tomohideshibata** [comment]
I want to run BERT (not Electra) pre-training. I would appreciate it if you could write a document on BERT pre-training.

Thanks in advance.

**Comment by stefan-it** [comment]
Hi @saberkun ,

is there any readme or pretraining command available for pretraining a BERT model? I have generated TFRecords available and used them with the Token Dropping implementation, which is working fine! However, I would also like to pretrain "normal" BERT models with this repo (instead of using the legacy 1.15 TensorFlow repo). Many thanks in advance!

**Comment by saberkun** [comment]
I updated with https://github.com/tensorflow/models/commit/bc8e6d24c21ed2939025a066095f3cab198547db. Could you help take a look and see what else you need?
Close for now. You can still leave comments or send a PR to improve the documentation.

**Comment by stefan-it** [comment]
Many thanks @saberkun ! I was particulary looking for the `wiki_books_pretrain.yaml` configiuration, and it is now available :hugs:

**Comment by stefan-it** [comment]
Hi @saberkun,

I have a follow-up question: I was able to sucessfully train various BERT models with Token Dropping approach. I would like to pretrain Large models now. Could you also please specify the hyper-parameters that are used for Large models? Many thanks again!
